roles/lib_openshift/src/ansible/oc_adm_registry.py

```diff
@@ -0,0 +1,47 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for registry
+    '''
+
+    module = AnsibleModule(
```

Comment:
@ashcrow, per @detiber's comments, you can pass in a --credentials or a serviceaccount since 3.2.  Jason mentioned we could just remove credentials in favor of a service account.  If that's not the case, I can re-add them.

@detiber, thoughts?